README.md

```diff
@@ -25,12 +25,46 @@ Finally, add `import SwiftKafka` to your source code.
 
 ### Producer API
 
+After creating the `KafkaProducer`, messages can be sent to a `topic` using the `send(_:)` method.
+
+```swift
+let config = KafkaProducerConfiguration(bootstrapServers: ["localhost:9092"])
+
+let producer = try KafkaProducer.makeProducer(
+    config: config,
+    logger: .kafkaTest // Your logger here
+)
+
+await withThrowingTaskGroup(of: Void.self) { group in
+
+    // Run Task
+    group.addTask {
+        try await producer.run()
+    }
+
+    // Task sending messages
+    group.addTask {
+        try producer.send(
+            KafkaProducerMessage(
+                topic: "topic-name",
+                value: "Hello, World!"
+            )
+        )
+
+        // Required
+        await producer.shutdownGracefully()
```

Comment:
We really want to get rid of this by folding it into the `run()` method. Maybe let's not add this to the example here.